FAQ.md

```diff
@@ -0,0 +1,22 @@
+| `could not open a connection to your authentication agent` | Pour vous identifier auprès d'un service tel que GitHub, vous tentez d'exporter votre clef SSH en l'ajoutant au cache de votre agent SSH avec : </br>`ssh-add ~/.ssh/fichier_ssh` | * S’assurer de l’exécution du ssh agent avec `eval $(ssh-agent)` </br></br>Et/OU </br></br>* S’assurer que l’adresse du dépôt n'est pas en HTTPS car elle doit être en SSH : `git remote -v`. </br>Le résultat attendu pour le dépôt tutorial est : </br>`origin	git@github.com:openfisca/tutorial.git (fetch)` </br>`origin	git@github.com:openfisca/tutorial.git (push)` </br>Pour en savoir plus, voir l'[aide GitHub](https://help.github.com/articles/changing-a-remote-s-url/#switching-remote-urls-from-ssh-to-https). |
+
+## Python OpenFisca
+
+| ERREUR | CONTEXTE | SOLUTION      |
+| ------ | -------- | ------------- |
+| `command not found` | Vous tentez d'appeler l'interprète du langage Python avec : `python un_fichier.py` | Mettre à jour la variable d'environnement `PATH` : </br>* Vérifier sa valeur avec : </br>`echo $PATH` </br>* Puis, retrouvez le répertoire où la version souhaitée de Python a été installée ; celui-ci varie d'un environnement à l'autre. </br>Exemple Windows : `/c/Python27`. </br>Exemple Unix : `/opt/local/bin/python` </br>* Mettre à jour la variable en indiquant le chemin vers l'interprète python. </br>Exemple Windows : `export PATH=/c/Python27:$PATH`. </br>Exemple Unix : `export PATH=/opt/local/bin:$PATH` </br></br>Afin que cette mise à jour soit permanente, il est conseillé d'ajouter cette commande d'export à la fin de votre fichier de configuration Bash `~/.bashrc`. |
+| `ImportError: No module named openfisca_france` | Vous tentez d'exécuter un fichier python tel que `framework.py` de ce dépôt tutorial avec : </br>`python framework.py` </br></br>Cela alors que l'environnement fait bien référence à OpenFisca-France lorsque vous le vérifiez avec `pip list`. | Sortir de votre environnement virtuel (`exit`). En créer un nouveau avec `pew new nom_de_l_environnement` </br>et démarrer une nouvelle installation d'OpenFisca-France avec `pip install openfisca-france` |
+| Vous n'appelez pas la version souhaitée de l'interprète Python | Vous avez installé la version de python que vous souhaitez utiliser mais dans votre environnement virtuel `python --version` indique une autre version. | Sortir de votre environnement virtuel (`exit`) et le redémarrer en indiquant la version de python souhaitée. Exemple pour la version 2.7 : </br>`pew workon nom_de_l_environnement --python=python27` |
```

Comment:
What happens when they don't have the right version installed on their machine ?

Reply:
Lien vers le site python ajouté.